### PR TITLE
first pass at builder-base install of EKS-D built go 1.15.15

### PR DESCRIPTION
**Pull request by danbudris**
*Issue #, if available:*
https://github.com/aws/eks-distro-internal/issues/206

*Description of changes:*
A conversation starter about how we want to perform the install of the patched 1.15 golang version from the RPMs we've generated.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: builder-base/install.sh [hunk 1]
```diff
@@ -69,8 +69,21 @@ function build::go::install(){
...
     fi
-    
-    if [[ $yum_provided_versions =~ (^|[[:space:]])${version%.*}($|[[:space:]]) ]]; then
+
+    local eks_built_versions="1.15"
+    if [[ $eks_built_versions =~ (^|[[:space:]])${version%.*}($|[[:space:]]) && $TARGETARCH == "amd64" ]]; then
+        local artifacts_bucket='eks-d-postsubmit-artifacts'
+        local arch='x86_64'
+        for artifact in golang golang-bin golang-race; do
+          aws s3 cp s3://$artifacts_bucket/golang/go/go1.15.15/RPMS/$arch/$artifact-1.15.15-1.amzn2.0.1.$arch.rpm /tmp
```
Comment (jaxesn) [inline builder-base/install.sh:78]: lets configure this specific version, probably down below where we call this function, pass in the full al2 golang version?

**Comment by danbudris** [comment]
hold until following prod 1.24 release

**Comment by danbudris** [comment]
superseded by https://github.com/aws/eks-distro-build-tooling/pull/494

**Pull request closed without merging**